### ISSUE TITLE
isMultiline semantics flag is not in sync with different platforms

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#34190]
Web introduce a new semantics flag ```_kIsMultilineIndex``` that is used for determine a text field into textarea or input text.

This flag is currently only exist in https://github.com/flutter/engine/blob/2d0103a8fe884ff8a30ddf38cf1157e560c1b9da/lib/stub_ui/lib/src/ui/semantics.dart#L300

This might cause confusion, we should copy this flag to other platforms as well

**Comment by chunhtai** [comment]
@jonahwilliams

**Comment by jonahwilliams** [comment]
+ @yjbanov

**Comment by yjbanov** [comment]
@mdebbar, should this be cleaned up or upstreamed?

**Comment by mdebbar** [comment]
This should be upstreamed. Without this flag, we can't support multiline text fields for semantics.

**Comment by mdebbar** [comment]
@nturgut is working on this [here](https://github.com/flutter/flutter/issues/34265).
